Userland/Libraries/LibCore/Event.h

```diff
@@ -66,17 +66,18 @@ class DeferredInvocationEvent : public Event {
 
 class TimerEvent final : public Event {
 public:
-    explicit TimerEvent(int timer_id)
+    using TimerId = int;
```

Comment:
These new IDs you've created would be better as `AK_TYPEDEF_DISTINCT_ORDERED_ID`s. from AK/DistinctNumeric.h. That way they cannot alias or convert with each other.